pkg/kubectl/util/hash/hash_test.go

```diff
@@ -148,7 +148,7 @@ not their metadata (e.g. the Data of a ConfigMap, but nothing in ObjectMeta).
 		obj      interface{}
 		expect   int
 	}{
-		{"ConfigMap", v1.ConfigMap{}, 3},
+		{"ConfigMap", v1.ConfigMap{}, 4},
```

Comment:
Shouldn't you have also updated the hash function to include the new binary data field in the hash? @liggitt @dims